crates/sui-graphql-rpc/src/server/builder.rs

```diff
@@ -684,7 +686,9 @@ pub mod tests {
                 })
                 .build_schema();
 
-            schema.execute("{ chainIdentifier }").await
+            schema
```

Comment:
I changed this test to rely on a db-backed query, because `chainIdentifier` is not a db thing anymore!

Reply:
Was this test failing before? It shouldn't actually depend on making a real DB call, because the delay we're testing for is simulated using a special GraphQL extension. There may be other tests where you need to do this though!

Reply:
Interesting. That timeout is used in two places: As an overall timeout for the request, and then as a DB request timeout (so that a long running request doesn't hang around even though the GraphQL request it is for has gone away).

In the test set-up, we introduce an extension to artificially extend the overall request time, but don't really do anything to intentionally extend the DB request time. I did notice that the test sets the same timeout as the delay we are introducing, so it's possible that before we were implicitly relying on the DB request to extend runtime. If that's the case, I think the simplest fix is to make sure we set the timeout clearly lower than the delay we're introducing (say -- `delay / 2`?)